frontend/src/components/Button/Button.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
+import './Button.scss';
```

Comment:
I don't think this is required Since it's been accounted for in `the index.scss`(entry file CSS)

Reply:
Index.scss should only have global styles. The button styles should not be global (and therefore should not be in index.scss) this keeps the styles modular and makes the user only download the styles they need for the particular page they are on right now.

Reply:
Where ever there are component styles that are in the index.scss they should not be there and I am removing them from there to keep these modular. That's why I added them in here and removed them from the index.scss

Reply:
Okay. I guess I might have to raised a PR for these changes too